dashboard_description_plugin/src/dp_dashboard_description_plugin/components/KdDescription.tsx

```diff
@@ -0,0 +1,146 @@
+// (C) 2022-2023 GoodData Corporation
+import React, { useMemo } from "react";
+import cx from "classnames";
+import invariant from "ts-invariant";
+import ReactMarkdown, { Options } from "react-markdown";
+import { CustomDashboardWidgetComponent, ICustomWidget } from "@gooddata/sdk-ui-dashboard";
+import { isIdentifierRef, isObjRef, ObjRef } from "@gooddata/sdk-model";
+
+import "./kdDescription.css";
+
+import {
+    DescriptionWidgetExtras,
+    PlaceholderPrefix,
+    ValueConfig,
+    ValuePlaceholderInDescription,
+} from "../interface.js";
+import { useAttributes } from "../hooks/useAttributes.js";
+
+import { InlineMeasure } from "./InlineMeasure.js";
+import { InlineAttribute } from "./InlineAttribute.js";
+import { getAttributeValue, getMetricRefFromConfig, mergeMessages } from "./utils.js";
+
+/**
+ * Dashboard widget that utilizes ReactMarkdown to render a reach text
```

Comment:
@dremileta render a **rich** text..

Reply:
Also bring the component annotations to the internal repo (first ideally next time) not to lose them here on a next update

Reply:
@dremileta I think here this is the last change needed (just the reach text type), otherwise we're good to merge, and then bring the changes to internal repo pls.

Reply:
@PavelJiranek done...and yeah, I already brought the changes to internal first